src/Network/Network/Generated/Models/PSApplicationGatewayAvailableServerVariableAndRequestHeaderResult.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <auto-generated>
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+// 
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+// 
+// For documentation on code generator please visit
+//   https://aka.ms/nrp-code-generation
+// Please contact wanrpdev@microsoft.com if you need to make changes to this file.
+// </auto-generated>
+
+using Microsoft.Azure.Management.Network.Models;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public partial class PSApplicationGatewayAvailableServerVariableAndRequestHeaderResult
+    {
+        public Dictionary<string, List<string>> Value { get; set; }
```

Comment:
I think having concrete properties would be better, but I'm not too sure.
```c#
public IList<string> AvailableServerVariables { get; set }
public IList<string> AvailableRequestHeaders { get; set }
public IList<string> AvailableResponseHeaders { get; set }
```

@cormacpayne what's your opinion?

Reply:
@nirkum18 @number213 concrete properties, as modeled above, would be better -- it's one less hop users have to make in order to get the information they want from this returned object.

Reply:
Updated